og_context/og_context.module

```diff
@@ -216,7 +216,7 @@ function og_context_is_init($called = FALSE) {
  *   Array keyed by the group type and group ID, or FALSE if no context
  *   was found.
  */
-function og_context($group_type = 'node', $group = NULL, $account = NULL, $check_access = TRUE) {
+function og_context($group_type = 'node', $group = NULL, $account = NULL, $check_access = FALSE) {
```

Comment:
I'd be against that change, since I'd think that OG should take a restrictive approach.

If my og_context() returns an unpublished node, I'd argue that 99% of the time, this is not the context you'd want to show. What do you think?

Reply:
I will agree!

Restrictive approach over breaking one function.
I think we already agreed on that when this function call was amended in og_context views handler https://github.com/Gizra/og/pull/131 https://github.com/Gizra/og/pull/131/files

All we need to do is let people know they have to amend their og_context() function call to make it work "the old way" and close all related issues as "Works as designed"
https://github.com/Gizra/og/issues/121